baker/algolia/utils/pages.ts

```diff
@@ -213,6 +215,26 @@ function getExcerptFromGdoc(
     }
 }
 
+function formatGdocMarkdown(content: string): string {
+    const simplifiedMarkdown = stripCustomMarkdownComponents(content)
+
+    const withoutGdocLinks = simplifiedMarkdown.replaceAll(
+        new RegExp(gdocUrlRegex, "g"),
+        ""
+    )
+
+    // There's a bug somewhere in enrichedToMarkdown that leaves "undefined" in the text
```

Comment:
This should probably be fixed at the source and with a migration that regenerates all the markdown columns. I just wanted to get a version of this "done" before finishing for the day.